scripts/kitten-tts/v0_8/run.sh

```
@@ -0,0 +1,69 @@
+    output_name=model.fp32.onnx
+    ;;
+  *)
+    echo "Unsupported KittenTTS v0.8 model: ${model_name}"
+    exit 1
+    ;;
+esac
+
+base_url=https://huggingface.co/${model_name}/resolve/main
+
+if [ ! -f ${onnx_name} ]; then
+  curl -SL -O ${base_url}/${onnx_name}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)